src/PrefsManager.cpp

```diff
@@ -213,6 +213,7 @@ PrefsManager::PrefsManager() :
 	m_AllowW1			( "AllowW1",			ALLOW_W1_EVERYWHERE ),
 	m_bEventMode			( "EventMode",			true ),
 	m_iCoinsPerCredit		( "CoinsPerCredit",		1 ),
+	m_iMaxNumCredits		( "MaxNumCredits",		40 ),
```

Comment:
Default should probably still stay 20 though.

Reply:
Should be set now @concubidated